package.json

```diff
@@ -30,8 +31,10 @@
     "test:unit:mocha": "mocha './app/**/*.test.js'",
     "test:e2e:chrome": "SELENIUM_BROWSER=chrome node test/e2e/run-all.js",
     "test:e2e:chrome:metrics": "SELENIUM_BROWSER=chrome node test/e2e/run-e2e-test.js test/e2e/metrics.spec.js",
+    "test:e2e:chrome:snaps": "SELENIUM_BROWSER=chrome node test/e2e/run-snaps.js",
```

Comment:
Rather than duplicating the `run-e2e-test.js` CLI, maybe we could update `run-all.js` to accept a test directory parameter? That would let us run just the snaps directory, which is I assume why you wrote `run-snaps.js`.